docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -471,7 +470,46 @@ To encode a raw image:
    ```
 
 <a name="animation"></a>
-## Encoding an Animated GIF
+## Encoding an Animated GIF or WEBP
+
+To encode an animated GIF or WEBP image:
```

Comment:
```suggestion 
To encode an animated GIF or WEBP image, follow these steps:

Reply:
ok! thank you for the comment.
I updated it.